tests/StacktraceTest.php

```diff
@@ -145,6 +145,34 @@ public function testAddFrameMarksAsInApp(): void
         $this->assertTrue($frames[2]->isInApp());
     }
 
+    public function testAddFrameMarksAsInAppOverrideExcludedPaths(): void
+    {
+        $this->options->setProjectRoot('path/to');
+
+        $this->options->setInAppExcludedPaths([
+                'path/to/excluded/folder',
+                'path/to/excluded/path',
+                'path/to/recursive/excluded/path',
+        ]);
+        $this->options->setInAppIncludedPaths([
+                'path/to/excluded/path',
+                'path/to/recursive/', //recursive
+        ]);
+
+        $stacktrace = new Stacktrace($this->options, $this->serializer, $this->representationSerializer);
+
+        $stacktrace->addFrame('path/to/excluded/path/file.php', 12, ['function' => 'test_function']);
+        $stacktrace->addFrame('path/to/recursive/file.php', 12, ['function' => 'test_function']);
+        $stacktrace->addFrame('path/to/recursive/excluded/path/file.php', 12, ['function' => 'test_function']);
+        $stacktrace->addFrame('path/to/excluded/folder/file.php', 12, ['function' => 'test_function']);
```

Comment:
We have one missing edge case here: paths which are included and NOT under the project root.

Reply:
What is the expected behavior here?
Because of the changes in (#911) all paths outside the project root getting excluded, even if i add them into _in_app_include_. 
If a path is specified in _in_app_include_, should it also get marked as _in_app_ if the path is outside of the project root?

Reply:
According to the documentation `in_app_include` takes precedence over `in_app_excluded`. `in_app` should became an entry of the `in_app_include` option, so the code should be adjusted accordingly to work in the way described before